src/components/Inputs/Input.jsx

```diff
@@ -3,7 +3,7 @@ import TextField from '@mui/material/TextField';
 
 import { useStyles } from './InputStyle';
 
-export const Input = ({ label, type = 'text', value, setValue, fullWidth }) => {
+export const Input = ({ label, type = 'text', value, setValue, fullWidth, helperText, error, required = false }) => {
```

Comment:
Only required is enough

Reply:
Earlier I did that, but in this case, for every Input, I have "*" event the input is not required. I do not why it happens that.
![asterix](https://user-images.githubusercontent.com/30056795/152297204-419160b6-0770-416f-8406-43ab97cbd0ae.png)

![Screen Shot 2022-02-03 at 8 10 08 AM](https://user-images.githubusercontent.com/30056795/152297754-7ed305c5-abcd-422e-a0ea-e912f48a8536.png)

If I set up Input as false 
```
  <Input label="Breed" type="text" value={breed} setValue={setBreed} required={false} />
```
also I have 

![star](https://user-images.githubusercontent.com/30056795/152299203-eb991969-d4fd-42ed-8afc-03e2f350fa1c.png)